serverless/networking/serverless-ossm-tls.adoc

```diff
@@ -0,0 +1,31 @@
+include::modules/serverless-document-attributes.adoc[]
+include::modules/ossm-document-attributes.adoc[]
+[id="serverless-ossm-tls"]
+= Configuring TLS for a custom domain using {ProductName}
+:context: serverless-ossm-tls
+include::modules/common-attributes.adoc[]
+
+toc::[]
+
+You can create a TLS key and certificates for a custom domain and subdomain using {ProductName}.
+
+[id="serverless-ossm-tls-prereqs"]
+== Prerequisites
+
+* Install xref:../../serverless/installing_serverless/installing-openshift-serverless.adoc#installing-openshift-serverless[{ServerlessProductName}].
+* Install {ProductName} xref:../../service_mesh/v1x/installing-ossm.adoc#installing-ossm-v1x[1.x] or xref:../../service_mesh/v2x/installing-ossm.adoc#installing-ossm-v2x[2.x].
+* Configure a custom domain. See xref:../../serverless/networking/serverless-ossm-custom-domains.adoc#serverless-ossm-custom-domains[Using custom domains for Knative services with Service Mesh].
+* In this example, `openssl` is used to generate certificates, but you can use any certificate generation tool to create these.
+
+[IMPORTANT]
+====
+This example uses the `example.com` domain.
+The example certificate for this domain is used as a certificate authority (CA) that signs the sub-domain certificate.
+
+To complete and verify this procedure in your deployment, you will need either a certificate signed by a widely trusted public CA, or a CA provided by your organization.
+
+Example commands should be adjusted according to your domain, sub-domain and CA.
+====
+
+include::modules/serverless-ossm-v1x-tls.adoc[leveloffset=+1]
+include::modules/serverless-ossm-v2x-tls.adoc[leveloffset=+1]
```

Comment:
As automatic updates are on by default, most users of SM will get upgraded to 2.x version of SM operator (even in 1.0 channel).

![Zrzut ekranu z 2021-01-15 12-22-03](https://user-images.githubusercontent.com/540893/104719192-67685380-572c-11eb-839a-c6c30588ae42.png)

New users will also get 2.x SM operator.

That's why I think we should list 2.x docs first.

```suggestion
include::modules/serverless-ossm-v2x-tls.adoc[leveloffset=+1]
include::modules/serverless-ossm-v1x-tls.adoc[leveloffset=+1]
```